source/administration/object-management/object-lifecycle-management.rst

```diff
@@ -63,77 +68,54 @@ Availability of Remote Data
+MinIO has two specific default behaviors for versioned buckets:
 
-- MinIO applies the expiration option to only the *current* object version by
-  creating a ``DeleteMarker`` as is normal with versioned delete.
+- MinIO applies the expiration option to only the *current* object version by creating a ``DeleteMarker`` as is normal with versioned delete.
 
-  To expire noncurrent object versions, specify the 
-  :mc-cmd:`~mc ilm add --noncurrentversion-expiration-days` option
-  when creating the expiration rule. 
+  To expire noncurrent object versions, specify the :mc-cmd:`~mc ilm add --noncurrentversion-expiration-days` option when creating the expiration rule. 
 
-- MinIO does not expire ``DeleteMarkers`` *even if* no other versions of 
-  that object exist.
+- MinIO does not expire ``DeleteMarkers`` *even if* no other versions of that object exist.
 
-  To expire delete markers when there are no remaining versions for that
-  object, specify the :mc-cmd:`~mc ilm add --expired-object-delete-marker`
-  option when creating the expiration rule.
+  To expire delete markers when there are no remaining versions for that object, specify the :mc-cmd:`~mc ilm add --expired-object-delete-marker` option when creating the expiration rule.
 
 .. _minio-lifecycle-management-scanner:
 
 Lifecycle Management Object Scanner
 -----------------------------------
 
-MinIO uses a built-in scanner to actively check objects against all
-configured lifecycle management rules. The scanner is a low-priority process
-that yields to high IO workloads to prevent performance spikes triggered 
-by rule timing. The scanner may therefore not detect an object as eligible 
-for a configured transition or expiration lifecycle rule until *after*
-the lifecycle rule period has passed.
-
-Delayed application of lifecycle management rules is typically associated to
-limited node resources and cluster size. Scanner speed tends to slow as 
-clusters grow as more time is required to visit all buckets and objects. 
-This can be exacerbated if the cluster hardware is undersized for regular
-workloads, as the scanner will yield to high cluster load to avoid performance
-loss. Consider regularly checking cluster metrics, capacity, and resource
-usage to ensure the cluster hardware is scaling alongside cluster and workload
-growth.
+MinIO uses a built-in scanner to actively check objects against all configured lifecycle management rules. 
+The scanner is a low-priority process that yields to high IO workloads to prevent performance spikes triggered by rule timing. 
+The scanner may therefore not detect an object as eligible for a configured transition or expiration lifecycle rule until *after* the lifecycle rule period has passed.
+
+Delayed application of lifecycle management rules is typically associated to limited node resources and cluster size. 
+Scanner speed tends to slow as clusters grow as more time is required to visit all buckets and objects. 
```

Comment:
I think the second `as` should be `and`. Also, we don't need the passive here.
> Scanner speed tends to slow as clusters grow and require more time to visit all buckets and objects.

Reply:
TBH I think I originally wrote this and missed the point.

The scanner doesn't magically speed up when your cluster or workload grows in size. It will naturally take longer to visit everything if nothing else changes.

Doing a re-write.